modin/experimental/pandas/io.py

```diff
@@ -25,7 +25,7 @@
 from . import DataFrame
 from modin.config import IsExperimental, Engine
 from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
-from ...pandas import _update_engine
+from modin.pandas import _update_engine
```

Comment:
Does not apply to this pull request, I can make a separate one. BTW what are your thoughts on this change? I think the explicit is better than the implicit.

Reply:
We need to reach agreement on how many dots are allowed to use for importing objects. 1 or 2 is okay to me.

Reply:
https://github.com/modin-project/modin/issues/4250